mlrun/runtimes/constants.py

```diff
@@ -78,6 +78,15 @@ def terminal_states():
             RunStates.error,
         ]
 
+    @staticmethod
+    def non_terminal_states():
+        return [
+            RunStates.running,
+            RunStates.created,
+            RunStates.pending,
+            RunStates.unknown,
+        ]
```

Comment:
You can do all - terminal I think

Reply:
Not really, only if we convert it to a set and back to a list which doesn't look as nice IMHO, wdyt?:
```
list(set(RunStates.all()) - set(RunStates.terminal_states()))
```